application.lisp

```diff
@@ -1,6 +1,16 @@
 (in-package :aark)
 
-(defparameter +delay+ (/ 1000.0 30.0))	; fps
+(defparameter +delay+ (/ 1000.0 30.0)
+  "Frames per second.")
+
+(defparameter +root+ (asdf:system-source-directory :aark)
+  "Absolute project root path.")
+
+;; TODO: provide something like config package with resource path
+;;       mappings and path-related routines. Now I can only hardcode them.
+(defmacro -res- (resource-file-name)
+  "Creates absolute path to resource by it's local filename."
+  `(merge-pathnames ,resource-file-name +root+))
```

Comment:
If `res` intended to be a macro, why not actually merge pathnames in compile stage?
In this case, source code `(res "font3.png")` will expand to `(merge-pathnames "font3.png" +root+))`.
We claim that aark should be installed via quicklisp, so asdf should be able to find system-source-directory of aark in compile time too.
So I suggest to rewrite that macro as
```cl
(defmacro res (resource-file-name)
  "Creates absolute path to resource by it's local filename."
  (merge-pathnames resource-file-name +root+))
```

Reply:
Oops, of course.